install/0000_80_machine-config-operator_05_osimageurl.yaml

```diff
@@ -12,7 +12,7 @@ data:
   # This (will eventually) replace the below when https://github.com/openshift/enhancements/pull/1032
   # progresses towards the default.
   baseOperatingSystemContainer: "placeholder.url.oc.will.replace.this.org/placeholdernamespace:rhel-coreos-8"
-  #baseOperatingSystemExtensionsContainer: "placeholder.url.oc.will.replace.this.org/placeholdernamespace:rhel-coreos-8-extensions"
+  baseOperatingSystemExtensionsContainer: "placeholder.url.oc.will.replace.this.org/placeholdernamespace:rhel-coreos-8-extensions"
```

Comment:
nit:  maybe we can rename`baseOperatingSystemContainer` to `baseOSExtensionsContainer`  and `baseOperatingSystemContainer` to `baseOSContainer` .  It is also consistent with OSImageURL .

Reply:
I think as a general rule we should s/container/image/ - because it's relatively standard to have "container" be *a running instance*.

So perhaps `baseOSImage`?  I don't otherwise have a strong opinion on that vs e.g. `baseOperatingSystemImage`

Reply:
Though, it's tricky here because "image" can often mean virtual/physical machine image (AMI/ISO) versus "container image", and while it's usually clear from context, *our* use of the term image blurs those lines.  Sometimes we need to spell it out fully by saying "container image" which is wordy.

Reply:
yeah, baseOSImage/baseOSContainerImg/baseOSContainerImage would be more technically correct. Will leave it to John whatever he prefers among them :)

Reply:
I went with `baseOSContainerImage`, it's still shorter than what we had :smile: